apps/every.near/widget/Layout.jsx

```diff
@@ -0,0 +1,78 @@
+const StandardLayout = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+`;
+
+const SidebarLayout = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+
+  .main {
+    flex-grow: 1;
+  }
+`;
+
+const SplitLayout = styled.div`
+  display: flex;
+  width: 100%;
+
+  .children {
+    flex-grow: 1;
+    flex-basis: 0;
+  }
+`;
+
+const Layout = ({ variant, blocks, children, asideChildren }) => {
+  const { Header, Footer } = blocks;
+
+  Header = Header ? Header : () => <></>;
```

Comment:
I had made blocks so they were specific to each layout -- I've imagined it as : 

Layout gives you a blueprint and what "blocks" can fit in each of the sections. Then you provide the blocks you want to place in.

For example, standard layout uses Header and Footer blocks.

So rather than an "AsideChildren", I think this should just be "Sidebar" block